lib/BC/Util.cpp

```diff
@@ -1339,6 +1340,17 @@ MoveConstantIntoModule(llvm::Constant *c, llvm::Module *dest_module,
             moved_c = ret;
             return ret;
           }
+#else
+          // In LLVM 16, cast is the only unary constexpr.
```

Comment:
This is copy & paste, de-duplicate (as far as I can tell the first branch is here twice?).

Reply:
The difference is that there's a conditional wrapping this logic above. I could de-duplicate but it would involve having multiple `#if` checks since I'd have to wrap the condition and the `else` block. I thought it'd be better cleaner and easier to just duplicate it.

Either that or I could put this in a lambda and make the logic look like:
```c++
#if LLVM_VERSION_NUMBER < LLVM_VERSION(16, 0)
if (uop->isCast()) {
  return move_cast();
} else {
  auto ret = llvm::ConstantExpr::get(
    ce->getOpcode(),
    MoveConstantIntoModule(ce->getOperand(0), dest_module,
      value_map, type_map));
  moved_c = ret;
  return ret;
}
#else
return move_cast();
#endif
```